### ISSUE TITLE
[Accessibility, a11y] Color contrast ratio of keyboard focus while navigating quick picks is 1.28:1 which is less than 3:1

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#237198]
- VS Code Version: 1.97.0-insider
- OS Version: macOS Sequoia 15.2
- Color Theme: Dark+ (Default Dark+)

This issue is being escalated from here: https://github.com/microsoft/vscode-azurefunctions/issues/4331

## Steps to Reproduce:

1. Open command palette and observe keyboard focus on any quick pick.  Example:

![Image](https://github.com/user-attachments/assets/2341f188-cf61-4ca7-9ab6-91b80fd93646)

## Actual
Color contrast ratio of keyboard focus is 1.28:1 which is less than 3:1

## Expected
Color contrast ratio of keyboard focus should be greater than or equal to 3:1

**Comment by TylerLeonhardt** [comment]
You're contrasting between the blue of a "focused item" and the gray of a "not focused item"?

**Comment by MicroFish91** [comment]
Yup, that's correct

**Comment by PoojaNamde** [comment]
@MicroFish91  we have logged new bug https://github.com/microsoft/vscode/issues/246460 to track this issue. Could you please close this one.
